uplink_compat.h

```diff
@@ -98,6 +98,7 @@
 #define free_object_iterator uplink_free_object_iterator
 #define open_project uplink_open_project
 #define close_project uplink_close_project
+#define revoke_access uplink_revoke_access
```

Comment:
We shouldn't add to this file, please remove.

Reply:
Thanks, I've removed it. At first, I've added for consistency, but now I realized what was the purpose of this file.